models/index.js

```diff
@@ -9,12 +7,22 @@ const env = process.env.NODE_ENV || 'development';
 const config = require('../config/config.js')[env];
 
 const db = {};
-
 let sequelize;
+const sequelizeConfig = {
+  ...config, logging: env === 'development'
+};
 if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+  sequelize = new Sequelize(
```

Comment:
Why do we need to change this?

Reply:
I wasn't changed, it was refactored to prevent the logging of all the SQL query during testing. That way it is easier to trace error with minimal query logged in the terminal.